lib/link_expansion/rules.rb

```diff
@@ -113,9 +113,10 @@ def potential_expansion_fields(document_type)
   end
 
   def expand_fields(edition, link_type)
```

Comment:
The method above, `potential_expansion_fields`, will need to be modified to return a flattened list of fields in this case, i.e. `:details` should be returned in this list directly.